tests/index.html

```diff
@@ -10,9 +10,9 @@
     {{content-for "head"}}
     {{content-for "test-head"}}
 
-    <link rel="stylesheet" href="assets/vendor.css">
-    <link rel="stylesheet" href="assets/preprint-service.css">
-    <link rel="stylesheet" href="assets/test-support.css">
+    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
```

Comment:
Updates from ember-cli to get the tests working correctly. A large portion of the people developing on preprints were having issues because the ember-cli is outdated.